lib/interface.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "board.rb"
+
+class Interface
+  attr_accessor :std_in, :std_out
+  attr_reader :board
+
+  def initialize(std_in:, std_out:)
+    @std_out = std_out
+    @std_in = std_in
+    @board = Board.new.to_s
```

Comment:
The Interface knowing how to create a board feels a little out of place, since (I think) its job is input/output. What do you think about having a board instance passed into `initialize`?

Reply:
in2 it